### PR TITLE
Eventapp

### PR DESCRIPTION
**Pull request by kyakusahmed**

### CODE REVIEW

**Comment by mozzy22** [comment]
the first thing, whenever u push more than one commit s, before u make a PR , please merge them into one

File: index.html [hunk 1]
```diff
@@ -1,10 +1,75 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     
     </head>
     <body>
+            <div class="topnav" id="myTopnav">
+                    <a href="#home" class="active">Home</a>
+                    <a href="#news">News</a>
+                    <a href="#contact">Contact</a>
+                    <a href="#about">About</a>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```
Comment (mozzy22) [inline index.html]: @kyakusahmed , am seeing a jsvascript function "myFuction", wat do u use it for, we are currently doing html and css,

File: index.html [hunk 2]
```diff
@@ -1,10 +1,75 @@
...
+                      <h2>WHITE TALENT SHOW</h2>
+                      <img SRC="images/img.jpg" class="" alt="">
+                  
+                  
 
+                      <div class="col-md-4">
+                          <form action="post">
+                              <div class="group">
+                              <label>Name</label>
+                              <br>
```
Comment (mozzy22) [inline index.html]: yesterday they gave us a recommendation not to use "<br>"

Comment (mozzy22) [reply index.html]: can u instead use divs

File: index.html [hunk 3]
```diff
@@ -1,10 +1,75 @@
...
 
-    </body>
-</html>
\ No newline at end of file
+                              <div class="group">
+                              <label>Email</label>
+                              <br>
+                              <input type="text" name="email" placeholder="your email">
+                              </div>
+                              <br>
```
Comment (mozzy22) [inline index.html]: avoid using br

File: index.html [hunk 4]
```diff
@@ -1,10 +1,75 @@
...
+
+
+
+
+
+
+
+
+
+                  <script>
```
Comment (mozzy22) [inline index.html]: in case u want to use JS, please use an external file

File: index.html (hunk 2 above)
Comment (kyakusahmed) [reply index.html]: thanks

File: index.html (hunk 3 above)
Comment (kyakusahmed) [reply index.html]: copy that

**Pull request merged**